### DISCUSSION TITLE
Please remove the shake animation from the login screen.

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#235] [Ideas]
A shake animation to signify an incorrect password is a terrible and annoying invention by Apple. All it does to the user experience is rubbing it into the user's face. The user has enough reason to be annoyed after having entered the wrong password; there is no need to rub it into users' faces.

The red "Incorrect password" text is more than enough to communicate to the user that they entered the wrong password. There is no need for the password field to vibrate like an earthquake.

Let's do to the shake animation the same that needs to happen to the pull-to-refresh anti-feature on mobile apps. It is time to eradicate it.

**Comment by Secret-chest** [comment]
Having it for the screensaver is fine, but NOT the windowed prompt!

**Comment by anaximeno** [comment]
I like it, thanks to that I don't have to read what's said in the message and "instinctively" know that I wrote my password wrong.

**Reply by HT-7** [reply]
> I like it, thanks to that I don't have to read what's said in the message and "instinctively" know that I wrote my password wrong.

The best visual indicator is that you are still on the login screen.

You already know that you wrote your password wrong if you still see a login screen infront of you.

The shake animation adds no value. It just adds emphasis to an unwanted fact.
